src/Libraries/Microsoft.Extensions.AI.Abstractions/Microsoft.Extensions.AI.Abstractions.csproj

```diff
@@ -8,6 +8,7 @@
 
   <PropertyGroup>
     <Stage>preview</Stage>
+    <SuppressFinalPackageVersion>true</SuppressFinalPackageVersion>
```

Comment:
What does this do?

Reply:
Makes it such that the package version won't be stabilized even when the rest of the repo is going stable.

To pre-answer your follow-up question 😋, Yes ideally that was what setting Stage to preview should do, but I found that wasn't working and so this is the change that was made in the release branch in the meantime to make sure packages wouldn't ship stable. Now that this code will be in main and we are no longer on a time crunch, I'll investigate and remove the duplication after this goes in.

Reply:
Shouldn't we set it in ProjectStaging.targets - a single point instead of changing multiple files?

```diff
diff --git a/eng/MSBuild/ProjectStaging.targets b/eng/MSBuild/ProjectStaging.targets
index e3a89d0354..6db968b7fb 100644
--- a/eng/MSBuild/ProjectStaging.targets
+++ b/eng/MSBuild/ProjectStaging.targets
@@ -28,4 +28,10 @@
     <Description Condition="'$(Stage)' == 'obsolete'">Obsolete Package. $(Description)</Description>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <SuppressFinalPackageVersion>false</SuppressFinalPackageVersion>
+    <!-- Makes it such that the package version won't be stabilized even when the rest of the repo is going stable. -->
+    <SuppressFinalPackageVersion Condition="'$(Stage)' == 'dev' or '$(Stage)' == 'preview'">true</SuppressFinalPackageVersion>
+  </PropertyGroup>
+
 </Project>
```